.github/scripts/python.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
Please, add some minimal comments at this file header to describe its purpose... :-)

Reply:
Added a description.